### PR TITLE
Server injection window

### PR DESCRIPTION
**Pull request by alexi**

### CODE REVIEW

**Review by jonasalexander — DISMISSED** [review]
🪞!!

File: src/mirror/mirror.py [hunk 1]
```diff
@@ -50,6 +59,7 @@ def mirror(
     window_scaling_factor: float,
     left: int = 0,
 ):
+    print("scaling factor:", window_scaling_factor)
```
Comment (jonasalexander) [inline src/mirror/mirror.py]: remove?

File: src/data.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "username": "admin",
+    "post": "sic transit gloria mundi"
+}
```
Comment (jonasalexander) [inline src/data.json]: is this related?

Comment (alexi) [reply src/data.json]: no, this is from for another, secret pr 👀

**Review by jonasalexander — APPROVED** [review]
lgtm!
